src/containers/LearningResourcePage/components/LearningResourcePanels.jsx

```diff
@@ -75,45 +21,76 @@ const LearningResourcePanels = ({
   handleSubmit,
   ...rest
 }) => (
-  <Accordion openIndexes={['learning-resource-content']}>
-    {({ openIndexes, handleItemClick }) => (
+  <Accordion>
+    {({ handleItemClick }) => (
       <AccordionWrapper>
-        {panels.map(panel => {
-          if (panel.showPanel && !panel.showPanel(values, userAccess)) {
-            return null;
-          }
-          const hasError = panel.errorFields.some(field => !!errors[field]);
-          return (
-            <Fragment key={panel.id}>
-              <AccordionBar
-                panelId={panel.id}
-                ariaLabel={t(panel.title)}
-                onClick={() => handleItemClick(panel.id)}
-                hasError={hasError}
-                title={t(panel.title)}
-                isOpen={openIndexes.includes(panel.id)}
-              />
-              {openIndexes.includes(panel.id) && (
-                <AccordionPanel
-                  id={panel.id}
-                  hasError={hasError}
-                  isOpen={openIndexes.includes(panel.id)}>
-                  <div className={panel.className}>
-                    {panel.component({
-                      hasError,
-                      userAccess,
-                      values,
-                      closePanel: () => handleItemClick(panel.id),
-                      fetchSearchTags,
-                      handleSubmit,
-                      ...rest,
-                    })}
-                  </div>
-                </AccordionPanel>
-              )}
-            </Fragment>
-          );
-        })}
+        <AccordionSection
+          id={'learning-resource-content'}
+          title={t('form.contentSection')}
+          className={'u-4/6@desktop u-push-1/6@desktop'}
+          hasError={!!(errors.title || errors.introduction || errors.content)}
+          startOpen>
+          <LearningResourceContent userAccess={userAccess} handleSubmit={handleSubmit} {...rest} />
+        </AccordionSection>
+        {values.id && !!userAccess?.includes(TAXONOMY_WRITE_SCOPE) && (
+          <AccordionSection
+            id={'learning-resource-taxonomy'}
+            title={t('form.taxonomySection')}
+            className={'u-6/6'}>
+            <LearningResourceTaxonomy
+              closePanel={() => handleItemClick('topic-article-taxonomy')}
+              userAccess={userAccess}
+              {...rest}
```

Comment:
går det ann å bli kvitt rest her uten for mye hassle?

Reply:
Jeg håper det🤞 Skal se på det i dag!